include/git2/tree.h

```diff
@@ -381,7 +381,7 @@ GIT_EXTERN(int) git_treebuilder_write(
  *
  * @see git_treebuilder_write
  *
- * @param id Pointer to store the OID of the newly written tree
+ * @param oid Pointer to store the OID of the newly written tree
```

Comment:
Generally, we prefer `id` in things like `commit_id` or `tree_id`.  We're sort of mixed about it when we're just talking about _an_ id.  There are a significant number of `oid`s when we're not explicit about what kind of id.

Anyway, we might want to clarify that, but this needn't be the place.  I'll drop my patches for these two changes.